requirements.txt

```diff
@@ -0,0 +1,12 @@
+aspy.yaml==0.2.2
```

Comment:
Some of them are dependencies of others and do not need to be declared explicitly. It's a mess.